tutorials/docusaurus.config.js

```diff
@@ -218,6 +218,17 @@ module.exports = {
         ],
       },
     ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Deploy a Solidity Smart Contract Using Hardhat',
+        description:
+          'In this tutorial, you will learn how to set up Hardhat and use it to build, test and deploy a simple smart contract on ShimmerEVM.',
+        preview: '/shimmerevm-hardhat.jpg',
+        route: 'tutorials/shimmerevm-hardhat',
+        tags: ['text', 'shimmer', 'solidity', 'shimmerevm', 'hardhat', 'isc'],
```

Comment:
Then need to update `tags.json`.
We don't call it `iscp` anymore. 😅
And need to add `shimmerevm` and `hardhat` as tags.

Reply:
I know about ISCP. But you would need to update all tutorials. So just leave as is and we can update afterwards. It doesn't matter anyway as it just is an internal tag

You don't need to add shimmerevm and hardhat. This tags are only used for filtering and I'm not sure if we can fit shimmerevm and hardhat there 🤔.

(We could add shimmerevm to network, but not sure if I like that. May suggest it will only work on Shimmerevm network although it works on every chain 🤔 )

Reply:
But I would definitely add solidity

Reply:
Yes will leave iscp -> isc for a separate PR. Updated to iscp for now. 

hardhat and ShimmerEVM might be important filter items. Upto you how to include them. Feel free to edit the PR.

Reply:
Ok, then just leave them. Could you create an issue to add those? And maybe other tags you consider important to filter for?

Reply:
Opened Issue #912